packages/core-components/src/components/OverflowTooltip/OverflowTooltip.tsx

```diff
@@ -16,14 +17,11 @@
 
 import { makeStyles } from '@material-ui/core/styles';
 import Tooltip, { TooltipProps } from '@material-ui/core/Tooltip';
-import React, { useState } from 'react';
-import TextTruncate, { TextTruncateProps } from 'react-text-truncate';
-import { useIsMounted } from '@react-hookz/web';
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
 
 type Props = {
-  text: TextTruncateProps['text'];
-  line?: TextTruncateProps['line'];
```

Comment:
Hm you could keep the line count feature if you used the clamp css props https://stackoverflow.com/a/13924997

You can inject props into the styles with a function syntax, see the mui docs.


Would you be up for this? :)

Reply:
I will try but already line props does not work. Try bigger description does not truncate because props does not have line value.

Reply:
Because i add whitespace nowrap then there is no use of clamp. 
You check if you have time our i will add screenshot later